blockchain/block/header.go

```diff
@@ -95,6 +99,7 @@ func (h *Header) BlockHeaderCoreProto() *iotextypes.BlockHeaderCore {
 	if h.logsBloom != nil {
 		header.LogsBloom = h.logsBloom.Bytes()
 	}
+	header.NumActions = h.numActions;
```

Comment:
you'll need to update go.mod file to point to the new iotex-proto version with NumActions added in header definition, w/o it the code won't compile

Reply:
Will update go.mod file once this is merged: https://github.com/iotexproject/iotex-proto/pull/37.